src/openrct2-ui/windows/StaffList.cpp

```diff
@@ -347,7 +347,7 @@ static void window_staff_list_tooldown(rct_window* w, rct_widgetindex widgetInde
         if (footpathCoords.x == LOCATION_NULL)
             return;
 
-        bool isPatrolAreaSet = staff_is_patrol_area_set(200 + selectedPeepType, footpathCoords.x, footpathCoords.y);
+        bool isPatrolAreaSet = staff_is_patrol_area_set_for_type((STAFF_TYPE)selectedPeepType, footpathCoords);
```

Comment:
do a `static_cast<STAFF_TYPE>(` instead of the C style cast. Same goes for the other instances.